charts/postgres-operator/templates/clusterrole-postgres-pod.yaml

```diff
@@ -13,7 +13,11 @@ rules:
 - apiGroups:
   - ""
   resources:
+{{- if toString .Values.configGeneral.kubernetes_use_configmaps | eq "true" }}
+  - configmaps
+{{- else }}
   - endpoints
+{{- end }}
```

Comment:
I would recommend to at least allow `get endpoints` even in the case when configmaps are used.

Reply:
Can you explain why this would be helpful? - As far as I understand endpoints aren't used at all if `configGeneral.kubernetes_use_configmaps` is set.

Reply:
I have some ideas to bypass `kubernetes` service when talking with K8s API servers, that would require having access to `get endpoints`. It would actually good to check it is ever allowed on openshift, but I don't have access to any OCP setups.

Reply:
Changed the patch according to your suggestions.